types/msg_service.go

```diff
@@ -0,0 +1,43 @@
+package types
```

Comment:
This file contains a function used by baseapp and by modules. The best place i could think of it here (or in a `msg_service` subfolder). Any ohter ideas?

Reply:
I would rather not add this to `types` as it depends on both `codec` and `grpc`. How about in `types/module` as it will generally need to be reference by modules. Or something like `types/service`. I want to work towards making `types/` leaner.